pkg/storage/snapshot/snapshot.go

```diff
@@ -222,6 +222,11 @@ func (ctrl *VMSnapshotController) updateVMSnapshot(vmSnapshot *snapshotv1.Virtua
 		return 0, err
 	}
 
+	// Update "kubevirt_vmsnapshot_timestamp_seconds" metric if the snapshot succeeded
+	if vmSnapshot.Status != nil && vmSnapshot.Status.Phase == snapshotv1.Succeeded {
```

Comment:
I think a better place to update this is in updateSnapshotStatus where the succeeded phase is updated

Reply:
moved